src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/IMsBuildAccessor.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Build.Evaluation;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Build
+{
+    /// <summary>
+    /// Utility class for allowing for testing of code that needs to access the msbuild lock, and also be testable.
+    /// </summary>
+    internal interface IMsBuildAccessor
```

Comment:
It's cased "MSBuild" - but in this case, I think this should be called something else - we don't use "MSBuild" elsewhere in APIs, such as IProjectXmlAccessor or something.

Build is the wrong namespace - this has nothing to do with "build" per se.